cluster/juju/layers/kubernetes-worker/layer.yaml

```diff
@@ -1,12 +1,14 @@
 repo: https://github.com/kubernetes/kubernetes.git
 includes:
   - 'layer:basic'
+  - 'layer:debug'
   - 'layer:docker'
+  - 'layer:nagios'
   - 'layer:tls-client'
-  - 'layer:debug'
   - 'interface:http'
   - 'interface:kubernetes-cni'
   - 'interface:kube-dns'
+  - 'interface:nrpe-external-master'
```

Comment:
These explicit calls aren't needed as they are included in `layer:nagios` (which we should rename to `layer:external-nagios`, tbh.